pkg/lib/dialogs.jsx

```diff
@@ -107,7 +107,8 @@ export const WithDialogs = ({ children }) => {
 
     const Dialogs = {
         show: setDialog,
-        close: () => setDialog(null)
+        close: () => setDialog(null),
+        isActive: dialog !== null
```

Comment:
Are you sure that this works? This is only initialized once, ever. It would make more sense to me if that was a function, i.e. `() => dialog !== null`.

Reply:
It works fine in navigator, but I will change as it makes more sense :+1: